src/crimson/osd/pg.cc

```diff
@@ -313,6 +313,7 @@ ghobject_t PG::do_delete_work(ceph::os::Transaction &t,
 {
   // TODO
   shard_services.dec_pg_num();
+  return _next;
```

Comment:
OK, the interface got changed recently by 7f047005fc72e1f37a45cde2d742bb2eb1e62881.